types.txt

```diff
@@ -0,0 +1,3 @@
+types-PyYAML
+types-click
```

Comment:
I think this needs to be removed. These are the type annotations for the old click before the new click got inbuilt type annotations.